test/redis/store/factory_test.rb

```diff
@@ -208,6 +208,11 @@
         store.to_s.must_equal("Redis Client connected to 127.0.0.1:6379 against DB 0 with namespace theplaylist")
       end
 
+      it 'honors scheme (rediss:// vs. redis://)' do
+        store = Redis::Store::Factory.create "rediss://127.0.0.1:6380"
+        store.instance_variable_get(:@client).scheme.must_equal('rediss')
```

Comment:
can you add a few more assertions to this test ensuring that the URI scheme won't break down 

something like

```ruby
store = Redis::Store::Factory.create "//127.0.0.1:6380"
store.instance_variable_get(:@client).scheme.must_equal('redis')

store = Redis::Store::Factory.create
store.instance_variable_get(:@client).scheme.must_equal('redis')
```

I noticed that if you don't include "//" then `URI.parse` will throw an `InvalidURIError`, so I don't think we need to support that.